### PR TITLE
Refluffs the Dominian Bomber Jacket and Sweater

### PR DESCRIPTION
**Pull request by WickedCybs**
This PR refluffs the dominian bomber jacket and sweater, aligning them more-so into being indicative of Fisanduhian styles and fashions rather than with the Imperials. It also changes the descriptions a bit and adds in new fluff text, and puts them into their own unique loadout selections.

I discussed whether this would fit with Schwann and a few others, I got overall good reception so I'm tentatively putting this PR up. My reasoning here is that overall, Dominian styles trend towards a more colourful and extravagant fashion in-game. What with the cloaks, dresses, suits. Fisanduhians apply a bit more of a utilitarian dynamic I find, favouring less flashy and more rugged things. 

image

### CODE REVIEW

**Comment by WickedCybs** [comment]
!review

**Comment by WickedCybs** [comment]
PR should wait for Lore Approval too.

**Review by Geevies — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**
